### DISCUSSION TITLE
内网环境：TypeError: n.replaceAll is not a function

### DISCUSSION THREAD

**Discussion by lovebai** [AlistGo/alist#6398] [General]
我把Alist部署在内网环境，内网计算机比较多，有些浏览器比较旧，也做不到让每个人都去升级浏览器，按照https://github.com/alist-org/alist/discussions/1662 和 https://alist.nn.ci/faq/why.html#open-alist-prompt-system-error-typeerror-n-replaceall-is-not-a-function 方法，我把阿里的polyfill.js放到内网服务器下还是不能解决问题，请问各位老哥们有什么解决的办法。

**Comment by pongfcnkl** [comment]
自己手动搭建Polyfill.io 服务 不要直接下js文件，不然是无效的
https://gacfox.com/articles/3
https://github.com/polyfillpolyfill/polyfill-service
或者想办法把阿里的polyfill.js反代到内网

**Reply by lovebai** [reply]
感谢您的帮忙，我决定使用docker部署到内网服务器，反代阿里的方法实现不了

**Comment by lovebai** [comment]
> 自己手动搭建Polyfill.io 服务 不要直接下js文件，不然是无效的 https://gacfox.com/articles/3 https://github.com/polyfillpolyfill/polyfill-service 或者想办法把阿里的polyfill.js反代到内网
